src/test/scala/org/kibanaLoadTest/ingest/Main.scala

```diff
@@ -9,7 +9,9 @@ import org.kibanaLoadTest.helpers.Helper.{getLastReportPath, getTargetPath}
 
 object Main {
   def main(args: Array[String]): Unit = {
-    val host = System.getenv("HOST_FROM_VAULT")
+    val hostValue = System.getenv("HOST_FROM_VAULT")
+    val host =
+      if (hostValue.startsWith("http")) hostValue else "https://" + hostValue
```

Comment:
Oh I love this, `if` as an expression, not a statement.

Reply:
no ternary operator :)